Explorer/Assets/Editor/CloudBuild.cs

```diff
@@ -62,12 +65,59 @@ public static void PreExport()
 
         }
 
+        // Defined in the @T_MacOS/@T_Windows64 configurations in Unity Cloud
         [UsedImplicitly]
         public static void PostExport()
         {
             Debug.Log($"~~ {nameof(CloudBuild)} PostExport ~~");
         }
 
+        /// <summary>
+        /// Remove warnings from the build that are just clutter
+        /// We still want to maintain them outside of the build process
+        /// </summary>
+        private static void GenerateIgnoreWarningsFile()
+        {
+            // List of warning codes to ignore
+            string[] warningsToIgnore = {
+                "8618", // Nullable reference types
+                "8625", // Cannot convert null literal to non-nullable reference type
+                "8602", // Dereference of a possibly null reference
+                "8604", // Possible null reference argument
+                "8619", // Nullable value types
+                "8620", // Argument cannot be used for parameter due to differences in the nullability of reference types
+                "8603", // Possible null reference return
+                "8600", // Converting null literal or possible null value to non-nullable type
+                "8601", // Possible null reference assignment
+                "0649", // Field is never assigned to, and will always have its default value
+                "0414", // Field is assigned but its value is never used
+                "0168", // Variable is declared but never used
+                "0219"  // Variable is assigned but its value is never used
+            };
+
+            // Path to the Assets folder
+            string assetsPath = Application.dataPath;
+            string filePath = Path.Combine(assetsPath, "csc.rsp");
```

Comment:
Do we need to to refresh or recompile so the file is properly recognized by Unity?